package.json

```diff
@@ -1,79 +1,82 @@
 {
-  "name": "soap",
-  "version": "0.40.0",
-  "description": "A minimal node SOAP client",
-  "engines": {
-    "node": ">=10.0.0"
-  },
-  "author": "Vinay Pulim <v@pulim.com>",
-  "dependencies": {
-    "content-type-parser": "^1.0.2",
-    "debug": "^4.1.1",
-    "formidable": "^1.2.2",
-    "get-stream": "^6.0.0",
-    "httpntlm": "^1.5.2",
-    "lodash": "^4.17.19",
-    "request": ">=2.9.0",
-    "sax": ">=0.6",
-    "strip-bom": "^3.0.0",
-    "uuid": "^8.3.2",
-    "xml-crypto": "^2.1.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vpulim/node-soap.git"
-  },
-  "main": "./index.js",
-  "types": "./lib/soap.d.ts",
-  "directories": {
-    "lib": "./lib",
-    "test": "./test"
-  },
-  "scripts": {
-    "prepare": "npm run build",
-    "build": "tsc -p .",
-    "clean": "rm -rf lib",
-    "watch": "tsc -w -p .",
-    "lint": "tslint -p tsconfig.json",
-    "toc": "./node_modules/.bin/doctoc Readme.md --github --maxlevel 3",
-    "cover": "nyc --extension=.ts --reporter=lcov --reporter=html --reporter=text mocha --timeout 15000 --exit test/*-test.js test/security/*.js",
-    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
-    "docs": "typedoc --out docs",
-    "test": "mocha --timeout 15000 --bail --exit test/*-test.js test/security/*.js"
-  },
-  "keywords": [
-    "soap"
-  ],
-  "license": "MIT",
-  "devDependencies": {
-    "@types/debug": "^4.1.2",
-    "@types/express": "^4.16.1",
-    "@types/lodash": "^4.14.169",
-    "@types/node": "^11.15.54",
-    "@types/request": "^2.48.1",
-    "@types/sax": "^1.0.1",
-    "@types/uuid": "^8.3.0",
-    "body-parser": "^1.15.2",
-    "colors": "^1.3.3",
-    "coveralls": "^3.0.5",
-    "diff": "^4.0.1",
-    "doctoc": "^1.4.0",
-    "duplexer": "~0.1.1",
-    "express": "^4.16.4",
-    "finalhandler": "^1.1.1",
-    "glob": "^7.1.7",
-    "jshint": "^2.10.1",
-    "mocha": "^6.1.4",
-    "nyc": "^14.1.1",
-    "readable-stream": "~2.0.2",
-    "semver": "^5.6.0",
-    "serve-static": "^1.14.1",
-    "should": "^13.2.3",
-    "sinon": "^1.17.7",
-    "source-map-support": "^0.5.10",
-    "timekeeper": "^2.1.2",
-    "tslint": "^5.18.0",
-    "typedoc": "^0.20.36",
-    "typescript": "^3.3.3333"
+    "name": "soap",
```

Comment:
please revert the white space changes in this file.

Reply:
Hi!
You compared with a previous commit, it is already fixed in the master.
https://github.com/vpulim/node-soap/pull/1148/files